spec/requests/cases_controller_spec.rb

```diff
@@ -14,6 +14,7 @@
 
     it 'creates a new case object in Salesforce' do
       sign_in(admin)
+      Salesforce.seed_groups!
```

Comment:
Question: does this become part of some transaction because it is in a certain RSpec context? Just want to make sure these groups will be cleaned up after the test is run. 🙂

Reply:
Yes, it will be cleaned up. Even if it didn't, it won't affect much since we're not adding this groups in earlier while app start.